### DISCUSSION TITLE
Error at build Vercel - Type error: Cannot find module type: ../Footer and 'WorkerError'

### DISCUSSION THREAD

**Discussion by SolusRGB** [vercel/community#810] [Help]
Hi, so I was updating an old website I build a month or two ago, and now I am receiving this error now when I route the build to Vercel. 

<img width="1388" alt="Screenshot 2022-08-30 at 12 18 46" src="https://user-images.githubusercontent.com/94470768/187423576-d0a84a7a-22df-4e05-8367-167ba2148228.png">

This is my TS config

<img width="638" alt="Screenshot 2022-08-30 at 12 19 21" src="https://user-images.githubusercontent.com/94470768/187423626-dcb22724-bbec-4814-9524-405aabcbc5a7.png">

File structure 
<img width="200" alt="Screenshot 2022-08-30 at 12 22 37" src="https://user-images.githubusercontent.com/94470768/187424266-77d02ee3-bdab-4750-8fa0-627f0037280d.png">

On my local, I am not receiving any errors, I have deleted and reinstalled the lock file, node modules, and next to still have this issue at build. I tried changing the names of my components, nothing is working. 

Although it worked in the past, any help would be appreciated thank you.

**Comment by SolusRGB** [comment]
https://vercel.com/guides/how-do-i-resolve-a-module-not-found-error

I tried this

Now I receive this 
<img width="1324" alt="Screenshot 2022-08-30 at 12 34 30" src="https://user-images.githubusercontent.com/94470768/187426539-336ac983-23d6-4f06-9c39-7542b188d485.png">

**Comment by SolusRGB** [comment]
And now this error 
<img width="1385" alt="Screenshot 2022-08-30 at 12 58 18" src="https://user-images.githubusercontent.com/94470768/187430834-88d348b0-0496-4a8a-9bbb-7067648a1f1c.png">

**Comment by swarnava** [comment]
are you able to run `build` command locally?

**Comment by amyegan** [comment]
If you can share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I would be happy to dig into these errors for you!
